webapp/src/scss/_layout.scss

```diff
@@ -165,7 +165,7 @@ body {
     top: 0;
     z-index: $zindex-modal-backdrop;
     width: 100%;
-    height: 100%;
+    height: 66.5vh;
```

Comment:
@saurabh391  will this work if you resize the screen?

Reply:
Yes, AEN and made some changes for that.